src/main/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtils.java

```diff
@@ -0,0 +1,607 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.apache.commons.lang;
+
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.regex.Pattern;
+import java.util.stream.IntStream;
+
+@SuppressWarnings("unused")
+public class ApacheCommonsStringUtils {
+
+    private static class Abbreviate {
+        @BeforeTemplate
+        String before(String s1, int width) {
+            return StringUtils.abbreviate(s1, width);
+        }
+
+        @AfterTemplate
+        String after(String s, int width) {
+            return s.substring(0, Math.min(s.length(), width));
```

Comment:
I believe there are many overloads of `abbreviate()`. This particular one adds a trailing ellipsis. So you probably want something like:
```suggestion
            return s == null || s.length() <= width ? s : s.substring(0, width - 3) + "...";
```